src/mixer/state.ts

```diff
@@ -269,13 +344,25 @@ export const load = (
 
   const shouldResetTrim = getState().settings.resetTrimOnLoad;
 
-  dispatch(
-    mixerState.actions.loadItem({ player, item, resetTrim: shouldResetTrim })
-  );
+  // Can't really load a ghost. Unload instead.
```

Comment:
Is there a plausible scenario in which this could happen? If not, is it worth logging this, so we catch it?

Reply:
The case is if you try and select it while it's still saving the show plan with the new item. It can happen if you're quick enough...

Reply:
But if I were to load a ghost, then setting intros/outros/cues would rather break/become rather weird in those cases.

Reply:
Might it be easier if it just got ignored, instead of specifically unloading?

Reply:
Done